pkg/in/api.go

```diff
@@ -109,15 +109,15 @@ func parseIncident(input string) (*Incident, error) {
 		i.Service = "59"
 	case "Cyclamen IT Platform Local":
 		i.Service = "9"
-	case "I-LEAP":
+	case "PPPT - ILEAP":
 		i.Service = "58"
 	case "Semaphore":
 		i.Service = "45"
 	default:
 		i.Service = "65"
 	}
 
-	fmt.Printf("parsed incident: %v, status: %v, comment id: %v\n", i.Identifier, i.Status, i.CommentID)
+	fmt.Printf("parsed incident: %v, status: %v, comment id: %v\n", i.IntID, i.Status, i.CommentID)
```

Comment:
Could extend this log to state for which service, not required though

Reply:
good idea, added